registry/extension/oci/discover.go

```diff
@@ -26,7 +25,7 @@ func (th *extensionHandler) getExtensions(w http.ResponseWriter, r *http.Request
 	w.Header().Set("Content-Type", "application/json")
 
 	// get list of extension information seperated at the namespace level
-	enumeratedExtensions := extension.EnumerateRegistered(r.Context())
+	enumeratedExtensions := extension.EnumerateRegistered(*th.Context)
```

Comment:
Nit. Generally the reference to the struct tends to be an abbreviation of its name, can get go with eh instead of th

Reply:
renamed to 'eh'